meson.build

```diff
@@ -926,8 +926,9 @@ endif
 
 drm = {
     'name': 'drm',
-    'deps': dependency('libdrm', version: '>= 2.4.75', required: get_option('drm')),
+    'deps': dependency('libdrm', version: '>= 2.4.96', required: get_option('drm')),
```

Comment:
It's not nice to require nearly 2 years newer libdrm because you don't feel like checking which symbols were available at 2.4.75, or even simply skipping all of them.

It's just debugging feature, which didn't even prove its worth yet, and we already have a fallback which doesn't use these symbols anyway (useless, but it's there - return DRM_FORMAT_UNKNOWN).

We shouldn't raise the minimum requirement for this IMO.

Reply:
I'd be more inclined to remove the entire debugging thing at this point tbh. Trying to keep track of what formats are there at which version is nonsense (too bad there isn't a decent libdrm API for this).